app/_hub/kong-inc/jq/index.md

```diff
@@ -0,0 +1,101 @@
+        - `compact_output`: Returns output in a compact form without additional spacing,
+          and with each JSON object on a single line. Defaults to `true`. Set to `false` for "pretty" output.
+        - `raw_output`: Outputs as raw strings, not JSON quoted. Default is `false`.
+        - `join_output`: Similar to `raw_output` but does not output newline separators. Default is `false`.
+        - `ascii_output`: jq usually outputs non-ASCII Unicode codepoints as UTF-8, even if the input specified
+           them as escape sequences (like "\u03bc"). Using this option, you can force jq to produce pure ASCII
+           output, replacing every non-ASCII character with the equivalent escape sequence. Default is `false`.
+        - `sort_keys`: Outputs the fields of each object with the keys in sorted order. Default is `false`.
+    - name: request_if_media_type
+      required: false
+      datatype: array of strings
+      default: ["application/json"]
+      description: |
+        A list of media type strings, which **must** be present in the `Content-Type` header for the request program to run.
+    - name: response_jq_program
+      required: semi
+      datatype: string
+      description: |
+        The jq program to run on the response body. For example, `.[0] | { "X-Foo": .foo }`. 
+        Either `request_jq_program` or `response_jq_plugin` **must** be included in configuration.
+    - name: response_jq_program_options
+      required: false
+      datatype: record
+      description: |
+        Boolean option flags to modify the run behavior of the jq program run on the response body.
+        - `compact_output`: Returns output in a compact form without additional spacing,
+          and with each JSON object on a single line. Defaults to `true`. Set to `false` for "pretty" output.
+        - `raw_output`: Outputs as raw strings, not JSON quoted. Default is `false`.
+        - `join_output`: Similar to `raw_output` but does not output newline separators. Default is `false`.
+        - `ascii_output`: jq usually outputs non-ASCII Unicode codepoints as UTF-8, even if the input specified
+           them as escape sequences (like "\u03bc"). Using this option, you can force jq to produce pure ASCII
+           output, replacing every non-ASCII character with the equivalent escape sequence. Default is `false`.
+        - `sort_keys`: Outputs the fields of each object with the keys in sorted order. Default is `false`.
+    - name: response_if_media_type
+      required: false
+      datatype: array of strings
+      default: ["application/json"]
+      description: |
+        A list of media type strings, which **must** be present in the `Content-Type` header for the response program to run.
+    - name: response_if_status_code
+      required: false
+      datatype: array of integers
+      default: [200]
+      description: |
+        A list of HTTP response status codes which **must** be present for the response program to run.
```

Comment:
Same comment as for media types: one of the codes must be present, correct? Not all (I don't even know how all would happen).

Reply:
Yes, the user can configure multiple acceptable response status codes, and the program will only run if the actual response status code is found in that list.

Reply:
ok will reword

Reply:
Still not sure this is quite right - the "list of HTTP response status codes" can't be present, as the response can only have one status. 

```suggestion
        A list of HTTP response status codes, one of which **must** match the response status code for the response program to run.
```
That's more accurate, if a little repetitive! ;)